keyboards/coban/pad3a/keymaps/via/config.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2021 RyanDam (https://github.com/RyanDam)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define DYNAMIC_KEYMAP_LAYER_COUNT 1
```

Comment:
why only one layer?

Reply:
I thought this keyboard is very small (only 2 keys) and sacrifice one key dedicated to layer switching is a waste, hence only 1 layer.
But now I think leaving a choice for user is also good, so I updated the code to allow 2 layers (more than 2 layers is redundant because we only have 2 key)